motoman_gp25_support/urdf/gp25_macro.xacro

```diff
@@ -0,0 +1,165 @@
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_1_s.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_2_l">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/visual/gp25_link_2_l.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_2_l.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_3_u">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/visual/gp25_link_3_u.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_3_u.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_4_r">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/visual/gp25_link_4_r.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_4_r.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_5_b">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/visual/gp25_link_5_b.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_5_b.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_6_t">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/visual/gp25_link_6_t.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp25_support/meshes/collision/gp25_link_6_t.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}tool0"/>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint name="${prefix}joint_1_s" type="revolute">
+        <parent link="${prefix}base_link"/>
+        <child link="${prefix}link_1_s"/>
+        <origin xyz="0 0 0.505" rpy="0 0 0" />
+        <axis xyz="0 0 1" />
+        <limit lower="-3.1416" upper="3.1416" effort="3087" velocity="3.6651"/>
```

Comment:
The limit tags might be a little more readable (and accurate) if you used math expressions to evaluate degrees (i.e. `${pi}` or `${radians(90)}`, etc.). It might be worth changing this everywhere you use radian values

Reply:
Yes, I agree.  I'll make the change.

Reply:
:+1: to this suggestion.

I've done the same in 07b8fc0c (which is one of the commits in the HC10 PR).